bin/functions/git.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
```

Comment:
I'm not sure how I feel about adding custom git aliases to the CMR repo. These aren't CMR specific in any way, so seem like they don't belong. This is more for a person's own custom setup.